### DISCUSSION TITLE
Repository insight graphs on GitHub are not showing

### DISCUSSION THREAD

**Discussion by yuryu** [google/mozc#1341] [General]
## Description
The graphs on the project "Insights" page don't load for google/mozc.

Specifically, the following page get stuck with "Crunching the latest data, just for you. Hang tight…"
- https://github.com/google/mozc/graphs/contributors
- https://github.com/google/mozc/graphs/commit-activity
- https://github.com/google/mozc/graphs/code-frequency

Contacting GitHub support to solve this issue. They said certain big files under `data/` are causing slowdowns and suggested treating them as binary in `.gitattributes`.

**Comment by yukawa** [comment]
Converting this to a discussion for now, as the next action is a bit unclear.
